docs: A sentence does not make sense.

m-ahmadi opened the pull request:
The sentence on line 23 does not make sense, Since I didn't know what should be the correct words, I didn't bother with a correct commit message, so hopefully someone else will fix it.

IgorMinar commented on aio/content/guide/cli-builder.md:
```diff
@@ -20,7 +20,7 @@ A builder handler function receives two arguments; a set of input `options` (a J
 The separation of concerns here is the same as with [schematics](guide/glossary#schematic), which are used for other CLI commands that touch your code (such as `ng generate`).
 
-* Options are given by the CLI user, context is provided by and provides access to the CLI Builder API, and the developer provides the behavior.
+* Options are given by the CLI user, context provides access to the CLI Builder API, and the developer provides the behavior.
```
how about this:

```suggestion
* The `options` object is provided by the CLI user, while the `context` object is provided by the CLI Builder API.
```

m-ahmadi replied:
Yes, that's much better.

IgorMinar commented on aio/content/guide/cli-builder.md:
```diff
@@ -20,7 +20,7 @@ A builder handler function receives two arguments; a set of input `options` (a J
-* Options are given by the CLI user, context is provided by and provides access to the CLI Builder API, and the developer provides the behavior.
+* Options are given by the CLI user, context provides access to the CLI Builder API, and the developer provides the behavior.
 
 * The `BuilderContext` object provides access to the scheduling method, `BuilderContext.scheduleTarget()`. The scheduler executes the builder handler function with a given [target configuration](guide/glossary#target).
```
```suggestion
* In addition to the contextual information, the `context` object, which is an instance of the `BuilderContext`, also provides access to a scheduling method, `BuilderContext.scheduleTarget()`. The scheduler executes the builder handler function with a given [target configuration](guide/glossary#target).
```

m-ahmadi replied:
Same here, much clearer.

m-ahmadi replied:
Since the commit message is not right, I guess someone else will handle these changes. (I'm not sure what to do)

sonukapoor replied:
@m-ahmadi I see that you created the branch using the github editor. You can download the branch to your local make the changes, add a new commit, and push again. Let me know if you need help with this.

IgorMinar reviewed (dismissed):
thanks for the PR! I agree that this text is confusing.

But how do you feel about the following additional tweaks in addition to your change?

sonukapoor commented:
@m-ahmadi Can you please squash the commits into one and add a longer commit message to fix the listing issue? It has to be a minimum of 100 chars.

m-ahmadi committed:
docs: rephrased two sentences

This commit rephrases a confusing sentence, and also adds more info to
another sentence to be more understandable.

m-ahmadi commented:
> @m-ahmadi Can you please squash the commits into one and add a longer commit message to fix the listing issue? It has to be a minimum of 100 chars.

Hi @sonukapoor , I did those things. Is everything ok?

sonukapoor commented:
LGTM thanks!

petebacondarwin reviewed (approved):
Reviewed-for: global-docs-approvers
